docs/docs/concepts/low_level.md

```diff
@@ -322,6 +325,63 @@ def continue_to_jokes(state: OverallState):
 graph.add_conditional_edges("node_a", continue_to_jokes)
 ```
 
+## `Command`
+
```

Comment:
I think you forgot to explain why it helps to decorate functions with Command type hint

Reply:
ya did that in the how to -- will copy over